locales/fr.yaml

```diff
@@ -0,0 +1,553 @@
+
+  Dès que tu as vérifié que le paiement est arrivé, tu dois débloquer les sats.
+buyer_took_your_order: |
+  🤖 ID d'offre : #${orderId}
+
+  @${buyerUsername} a récupéré ton offre et veut acheter tes sats. Contacte-le pour lui indiquer comment t'envoyer ${fiatAmount} ${currency} via ${paymentMethod}.
+
+  Dès que tu as vérifié que le paiement est arrivé en totalité, tu dois débloquer les sats
+waiting_seller_to_pay: 'J''ai envoyé une requête de paiement au vendeur pour qu''il envoie tes sats concernant l''offre dont la référence est #${orderId}, dès que le paiement est fait je vous informerai tous les deux.'
+sell_success: Ta vente de sats a été effectuée avec succès suite à la confirmation de paiement de @${buyerUsername} ⚡️🍊⚡️
+funds_released: 🕐 @${sellerUsername} a déjà débloqué les sats, ta facture sera payée incessamment, rappelle-toi que ton portefeuille doit être connecté pour la réception via Lightning Network.
+rate_counterpart: 'Merci de nommer ta contre-partie:'
+cant_process_order: Cet offre ne peut pas être traité, merci de vérifier que l'ID d'offre est correct.
+cant_release_order: Cet offre ne peut pas être débloqué, l'acheteur n'a pas fourni une facture à laquelle envoyer le paiement
+no_id_related: Tu n'as aucun offre associé à cette référence.
+sell_sats: Vends des satoshis
+buy_sats: Achète des satoshis
+order_detail: |
+  ID : `${order._id}`
+
+  Statut : ${status}
+
+  Créateur : @${creator || ''}
+
+  Acheteur : @${buyerUsername || ''}
+
+  Réputation de l'acheteur : ${buyerReputation}
+
+  Vendeur : @${sellerUsername || ''}
+
+  Réputation du vendeur : ${sellerReputation}
+
+  Montant de l'offre \\(sats\\) : ${order.amount}
+
+  Frais du bot \\(sats\\) : ${fee}
+
+  Frais de routage \\(sats\\) : ${order.routing_fee}
+
+  Montant FIAT \\(fiat\\) ${order.fiat_code} : ${order.fiat_amount}
+
+  Mode de paiement de l'offre : ${paymentMethod}
+
+  Plus-value/remise : ${priceMargin}
+
+  Créé le : ${createdAt}
+
+  Pris à : ${takenAt || ''}
+
+  seller invoice hash: `${order.hash || ''}`
+
+  seller invoice secret: `${order.secret || ''}`
+
+  buyer payment request: `${order.buyer_invoice || ''}`
+seller: vendeur
+buyer: acheteur
+dispute_started_channel: |
+  L'utilisateur ${type} @${initiatorUser.username} 
+  à déclenché un litige avec @${counterPartyUser.username} concernant l'offre 
+
+  ${detailedOrder}
+
+  @${initiatorUser.username} a été impliqué.e dans ${initiatorUser.disputes} litiges
+  @${counterPartyUser.username} a été impliqué.e dans ${counterPartyUser.disputes} litiges
+you_started_dispute_to_buyer: 🥴 Tu as initié un litige à propos de ton offre d'achat, nous allons te contacter toi et ta contrepartie en vue de l'arbitrage.
+buyer_started_dispute_to_seller: '🥴 L''acheteur a initié un litige concernant l''offre ID: ${orderId}, nous allons te contacter toi et ta contrepartie en vue de résoudre le problème.'
```

Comment:
same thing mentioned above

this should be somehting like:

```
buyer_started_dispute_to_seller: "🥴 L'acheteur a initié un litige concernant l'offre ID: ${orderId}, nous allons te contacter toi et ta contrepartie en vue de résoudre le problème."
```

please do the same in all the cases where you have two single quotes one after the other like this `''`